antlr-kotlin-runtime/src/commonMain/kotlin/com/strumenta/antlrkotlin/runtime/CopyOnWriteArrayList.kt

```diff
@@ -5,4 +5,26 @@ package com.strumenta.antlrkotlin.runtime
 public expect class CopyOnWriteArrayList<E>() : MutableList<E> {
   // Convenience constructor to avoid initializing with mutable state
   public constructor(elements: Collection<E>)
+
+  override val size: Int
```

Comment:
Do you need to specify in the expect class that you are going to override these methods in the actual class?

Reply:
Yes that's a K2 requirement.